File: site/src/main/java/edu/ncrn/cornell/site/ServletInitializer.java
package edu.ncrn.cornell.site;

import edu.ncrn.cornell.Ced2arApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.context.web.SpringBootServletInitializer;

public class ServletInitializer extends SpringBootServletInitializer {

	@Override
	protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
		return application.sources(Ced2arApplication.class);
	}

}

File: site/src/main/java/edu/ncrn/cornell/site/ServletInitializer.scala
package edu.ncrn.cornell.site

import edu.ncrn.cornell.Ced2arApplication
import org.springframework.boot.builder.SpringApplicationBuilder
import org.springframework.boot.web.support.SpringBootServletInitializer

class ServletInitializer extends SpringBootServletInitializer {
  override protected def configure(application: SpringApplicationBuilder): SpringApplicationBuilder =
    application.sources(classOf[Ced2arApplication])

}